conf/xml/Human_Dynamics_from_remapper.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+<robot name="Human-HDE" build=0 portprefix="">
+
+    <device type="ihuman_state_remapper" name="HumanStateRemapper">
```

Comment:
@lrapetti are you sure we need this configuration file? and has this been tested ?

I understand the idea we can simply use `HumanStateRemapper` in place of running `HumanStateProvider` device in the same configuration file. However, as we are currently using all the main devices in a single configuration file [Human.xml](https://github.com/robotology/human-dynamics-estimation/blob/devel/conf/xml/Human.xml), we can drop this file as maintaining multiple configuration files that are almost always similar is a bit hard. Instead, we can update [Human.xml](https://github.com/robotology/human-dynamics-estimation/blob/devel/conf/xml/Human.xml) with a comment that says `HumanStateRemapper` device also be used without having to run `HumanStateProvider` device.

Reply:
This was a configuration  file I was using for testing the device, and it is actually "per se" useless. I agree regarding the problem of maintaining multiple files, and I will remove it adding the comment to  `Human.xml`.

Reply:
Done https://github.com/robotology/human-dynamics-estimation/pull/204/commits/f294d7cbc5fac1d8300d7979f6cafe89d9322f6b